pkg/controllers/workspace_controller.go

```diff
@@ -471,6 +471,17 @@ func (c *WorkspaceReconciler) ensureNodePlugins(ctx context.Context, wObj *kaito
 	}
 }
 
+// getPresetName returns the preset name from wObj if available
+func getPresetName(wObj *kaitov1alpha1.Workspace) *string {
```

Comment:
"A string has two values in it: pointer to an array, and the string length. When you pass string as an argument, those two values are copied, not the underlying array." 

It is ok to return string type to simplify code, not much overhead.

Reply:
We return a pointer to perform nil check